docs/rfcs/XXXX_storage_backends.md

```diff
@@ -0,0 +1,248 @@
+To configure a storage backend, an organization administrator will be required to provide details about the AWS account where their target bucket resides and a user who may access it. An organization administrator will be able to configure this by navigating to the *Organization Settings* screen and choosing *Storages*. This will allow the administrator to add a new storage by first asking the user to create a new IAM user in their AWS account and assigning a specified policy. After the user has done this, they will be required to provide the AWS access and secret key pair so that Dewrangle may access the bucket. The final step will be to allow the user to specify the bucket and what prefix under it they would like this storage backend to use as the root. After confirming their choices, Dewrangle will attempt to access the bucket with the provided credentials and save the new storage backend if successful.
+
+### Use of Storage Backends
+
+A Storage Backend may be used in one of two ways: Either as a Document Storage Backend or a Data Storage Backend. Each method will require the same configuration of a Storage Backend and, in fact, a single Storage Backend may be used for both Documents and Data.
+
+#### Document Storage Backends
+
+Storage Backends used for documents will be read/write. Dewrangle will use these backends to save documents using a `uploads/` prefix under the root of the Storage Backend.
+
+#### Data Storage Backends
+
+Storage Backends used for data will only utilize read and list operations. These backends will be used for file discovery and accounting only.
+
+### Setting Organization Storage Backend Defaults
+
+If an organization administrator wishes all new studies to use a given backend for document uploads or data files, they may specify it in the organization settings. The administrator will have two defaults to configure: *Default Document Storage Backend* and *Default Data Storage Backend*. An administrator may use the same Storage Backend for both, but it will be up to them to prevent collisions.
+
+### Changing Storage Backends Per-Study
+
+It may be desirable for a study administrator to use different Storage Backends for different studies. For this, studies will be able to specify one Storage Backend for documents and one or more Storage Backends for data files. 
+
+#### Kids First Example
+
+In Kids First, each study keeps its data in different buckets including documents. If we were to use the Storage Backend approach for Kids First, we could do so by first creating a new Storage Backend for each bucket and then updating each study with the appropriate Storage Backend.
+If we would like to stop distributing uploaded documents to each study's bucket, we could simply update each study's Document Storage Backend to use the Default Storage Backend. This would result in all uploaded documents going to the application's bucket while all data operations occurring on each individual study's Data Storage Backend.
```

Comment:
```suggestion
If we would like to stop distributing uploaded documents to each study's bucket, we could simply update each study's Document Storage Backend to use the Default Storage Backend. This would result in all uploaded documents going to the application's bucket with all data operations occurring on each individual study's Data Storage Backend.
```